services/community-site/src/pages/Delegation/Delegation.tsx

```diff
@@ -35,15 +35,14 @@ interface Node {
   user: number;
   name: string;
```

Comment:
Can you move this interface to a new file so we can use it in the `EditNode` screen also?